### ISSUE TITLE
Solution for use in Rails Engine

### ISSUE THREAD

**Issue by pboling** [pantographe/view_component-form#100]

**Comment by nicolas-brousse** [comment]
Hi @pboling,

Sorry for the delay.
I'm currently working on a gem that have an engine. But I don't use `ActionText` on it.
I'll try to use it to see if I can reproduce what you described here.

**Comment by nicolas-brousse** [comment]
Hi @pboling, I did a draft PR https://github.com/pantographe/view_component-form/pull/109 to try to fix your issue. May you try in your app by loading the gem from the PR branch `fix-rich-text-component`? And tell me if it fix it or not.

Also to understand, in your app, how does rails loaded in `config/application.rb`? In regular way with `require 'rails/all'`?

**Comment by pboling** [comment]
@nicolas-brousse oooh, interesting.  We do require `rails/all` in our bin stubs within each engine, however, when loading/running/testing the main app, which loads those engines, and when testing the engines discretely, we do not use the bin stubs, we and *do not* require `rails/all`.  This is an inconsistency that I should sort out.

This is the main sequence:
```
# frozen_string_literal: true

require_relative "boot"

require "rails"
# Pick the frameworks you want:
require "active_model/railtie"
require "active_job/railtie"
require "active_record/railtie"
# require 'active_storage/engine'
require "action_controller/railtie"
require "action_mailer/railtie"
# require 'action_mailbox/engine'
# require 'action_text/engine' # <========================== It looks like this is the cause of our issue.
require "action_cable/engine"
require "rails/test_unit/railtie"

require "core"
require "csv"

# Require the gems listed in Gemfile, including any gems
# you've limited to :test, :development, or :production.
Bundler.require ...
```

**Comment by pboling** [comment]
The real bug is here https://github.com/pantographe/view_component-form/issues/111

**Comment by pboling** [comment]
Reopening, as this is still not fixed, and the solution in #111 is bad.  #109 is probably a better solution.
